src/mad_spark_alt/cli.py

```
@@ -40,6 +41,58 @@
 
 
+def _get_semantic_operator_status() -> str:
+    """Get status of semantic operators (ENABLED/DISABLED)."""
+    if LLMProvider.GOOGLE in llm_manager.providers:
+        return "Semantic operators: ENABLED"
+    else:
+        return "Semantic operators: DISABLED (traditional operators only)"
+
+
+def _format_idea_for_display(
+    content: str, max_length: int = 200, wrap_lines: bool = False
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)